src/libsync/propagateremotemkdir.cpp

```diff
@@ -122,11 +123,12 @@ void PropagateRemoteMkdir::slotMkdir()
     }
 
     const auto remoteParentPath = parentRec._e2eMangledName.isEmpty() ? parentPath : parentRec._e2eMangledName;
+    const auto absoluteRemoteParentPath = remoteParentPath.isEmpty() ? rootPath : rootPath + remoteParentPath + '/';
```

Comment:
Just a sidenote: There's some mixture of `'/'` vs. `QLatin1Char('/')` in all of those files ^^

Reply:
`rootPath` always has a trailing slash, correct?

Reply:
> `rootPath` always has a trailing slash, correct?

Yes

Reply:
> Just a sidenote: There's some mixture of `'/'` vs. `QLatin1Char('/')` in all of those files ^^

Yes, and we're also mixing "...", QStringLiteral("...") and QLatin1String("..."). This shall get proper cleanup PRs later on.